src/main/java/blusunrize/immersiveengineering/client/IEManual.java

```diff
@@ -124,14 +113,8 @@ public static void addIEManualEntries()
 					)
 			);
 			builder.readFromFile(new ResourceLocation(MODID, "thermoelectric"));
-			ieMan.addEntry(energyCat, builder.create());
+			ieMan.addEntry(energyCat, builder.create(), ieMan.atOffsetFrom(energyCat, "redstone_wire", 0.5));
```

Comment:
I appreciate the idea of using an "offset from" stategy, but what if someone completely replaces the manual's contents? Won't this throw an exception?

Reply:
> what if someone completely replaces the manual's contents?

How would they? I'm loading *all* files at that location in active resource packs, so the only reason why that entry would be missing would be if the IE resource pack is disabled. And I suspect that a lot of other things would break in that situation.

That has reminded me of an issue I need to look into though, there's no guarantee for the order of the `autoload` files from different packs as far as I know. So autoloaded entries from IE might end up below modded entries on one launch, but above on the next launch. The best approach would probably be to have a `autoloadPriority` or similar key at the top level of the JSON and load them in increasing order of that property.

Reply:
Ah, I missed that part then. I assumed a resource pack was overwriting the file, not appending it.